src/Samples/CodeEditorContribution/Contribution.d.ts

```diff
@@ -0,0 +1,340 @@
+import { GitVersionDescriptor, TfvcVersionDescriptor } from "azure-devops-extension-api/Git/Git";

```

Comment:
The interfaces here should be available to everyone in the azure-devops-extension-api library. I would put them in src/Git/CodeEditorTypes.d.ts

in https://github.com/Microsoft/azure-devops-extension-api